advanced/is-pattern-2/templates/analytics-worker/identity-server-analytics-worker-service.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-analytics-worker-service
+  namespace : {{ .Release.Namespace }}
+  labels:
+    app: is-pattern-2
+    monitoring: {{ .Values.wso2.monitoring.prometheus.jmxJobName }}
+    node: node-1
+spec:
+  clusterIP: None
+  selector:
+    deployment: {{ template "fullname" . }}-analytics-worker
+    node: node-1
```

Comment:
@darsham3 we don't need this label. Only `deployment: {{ template "fullname" . }}-analytics-worker` is enough when mapping to the service.